### PR TITLE
Update docs for release 0.6.0

### PR DESCRIPTION
**Pull request by ckadner**
Updated version references from `0.5.0` to `0.6.0` under the `sdk/` folder in preparation of the [`0.6.0` release on PyPI](https://test.pypi.org/project/kfp-tekton/0.6.0rc0/)

### Files modified:

- `guides/kfp_tekton_install.md`
- `sdk/python/kfp_tekton/__init__.py`
- `sdk/python/requirements.in`
- `sdk/python/setup.py`
- `Makefile`
- `VERSION`

/cc @Tomcli

### CODE REVIEW

File: guides/kfp_tekton_install.md [hunk 1]
```diff
@@ -55,16 +58,19 @@ To install the standalone Kubeflow Pipelines with Tekton, run the following step
 
 ## Kubeflow installation including Kubeflow Pipelines with Tekton Backend
 
-**Important: Please complete the [prequisites](#installation-targets-and-prequisites) before proceeding with the following instructions.**
+**Important: Please complete the [prerequisites](#installation-targets-and-prerequisites) before proceeding with the following instructions.**
 
-1. Follow the [Kubeflow install instructions](https://www.kubeflow.org/docs/ibm/deploy/install-kubeflow/#kubeflow-installation) to install whole Kubeflow stack with the kfp-tekton installation. Kubeflow v1.2.0 uses Tekton v0.14.0 and kfp-tekton v0.4.0.
+1. Follow the [Kubeflow install instructions](https://www.kubeflow.org/docs/ibm/deploy/install-kubeflow/#kubeflow-installation)
+   to install the entire Kubeflow stack with `kfp-tekton`.
+   Kubeflow `1.2.0` uses Tekton `v0.14.0` and `kfp-tekton` `v0.4.0`. <!-- TODO update-->
```
Comment (ckadner) [inline guides/kfp_tekton_install.md:65]: @Tomcli -- are we working on an updated version of this installation process that uses the latest version of Kubeflow, Tekton, KFP-Tekton?

Comment (Tomcli) [reply guides/kfp_tekton_install.md:65]: Kubeflow (full stack) 1.3 will be releasing at end of Q1/early Q2. We will update it when it's time for the new official release.

File: guides/kfp_tekton_install.md [hunk 2]
```diff
@@ -26,21 +26,24 @@ A Kubernetes cluster `v1.16` that has least 8 vCPU and 16 GB memory.
...
-1. Install [Tekton v0.16.3](https://github.com/tektoncd/pipeline/releases/tag/v0.16.3)
 
-2. Install Kubeflow Pipelines with Tekton backend (kfp-tekton) v0.5.0 [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)(CRDs).
+1. Install [Tekton v0.20.1](https://github.com/tektoncd/pipeline/releases/tag/v0.20.1)
+
+2. Install Kubeflow Pipelines with Tekton backend (`kfp-tekton`) `v0.6.0` [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)(CRDs).
    > Note: You can ignore the error `no matches for kind "Application" in version "app.k8s.io/v1beta1"` since it's a warning saying `application` CRD is not yet ready.
     ```shell
-    kubectl apply --selector kubeflow/crd-install=true -f install/v0.5.0/kfp-tekton.yaml
+    kubectl apply --selector kubeflow/crd-install=true -f install/v0.6.0/kfp-tekton.yaml
```
Comment (Tomcli) [inline guides/kfp_tekton_install.md:41]: #434 should have the new manifests

File: guides/kfp_tekton_install.md (hunk 1 above)
Comment (ckadner) [reply guides/kfp_tekton_install.md:65]: Okay, should I revert this line back to `Kubeflow v1.2.0 uses Tekton v0.14.0 and kfp-tekton v0.4.0`?

File: guides/kfp_tekton_install.md (hunk 2 above)
Comment (ckadner) [reply guides/kfp_tekton_install.md:41]: great. thanks Tommy

File: guides/kfp_tekton_install.md (hunk 1 above)
Comment (Tomcli) [reply guides/kfp_tekton_install.md:65]: sure, but I think you didn't change any wording other than the newlines. so it looks good to me.

**Comment by ckadner** [comment]
FYI: The KFP-Tekton SDK `0.6.0` is now on PyPI: https://pypi.org/project/kfp-tekton/0.6.0/

```Bash
pip install kfp-tekton==0.6.0
```

**Pull request merged**